packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/NextButton/NextButton.js

```diff
@@ -1,98 +1,56 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import styled, { ThemeProvider } from 'styled-components'
-import theme from 'styled-theming'
-import { adjustHue } from 'polished'
 import counterpart from 'counterpart'
-import { Button, Text } from 'grommet'
-import { FormNextLink } from 'grommet-icons'
-import zooTheme from '@zooniverse/grommet-theme'
+import { Button, Text, ThemeContext } from 'grommet'
+import { bool, func } from 'prop-types'
+import React from 'react'
+
+import nextButtonTheme from './theme'
 import en from './locales/en'
 
 counterpart.registerTranslations('en', en)
 
-const gold = zooTheme.global.colors['neutral-4']
-
-// TODO: box-shadow, border-radius, and disabled likely can move into the grommet theme.
-export const StyledNextButton = styled(Button)`
-  background: ${theme('mode', {
-    dark: zooTheme.global.colors['dark-1'],
-    light: gold
-  })};
-  border: ${theme('mode', {
-    dark: `solid thin ${gold}`,
-    light: `solid thin ${gold}`
-  })};
-  border-radius: 0;
-  box-shadow: none;
-  color: ${theme('mode', {
-    dark: gold,
-    light: 'black'
-  })};
-  flex: 3 0 auto;
-  padding: 0.5em;
-  text-transform: capitalize;
-
-  svg {
-    fill: ${theme('mode', {
-    dark: gold,
-    light: 'black'
-  })};
-    stroke: ${theme('mode', {
-    dark: gold,
-    light: 'black'
-  })};
-  }
-
-  &:hover:not(:disabled), &:focus:not(:disabled) {
-    background: ${theme('mode', {
-    dark: gold,
-    light: adjustHue(-7, gold)
-  })};
-    border: ${theme('mode', {
-    dark: `solid thin ${gold}`,
-    light: `solid thin ${adjustHue(-7, gold)}`
-  })};
-    box-shadow: none;
-    color: ${theme('mode', {
-    dark: 'black',
-    light: 'black'
-  })};
-  }
+// TODO: This will need hooking up to counterpart / a helper function to
+// determine whether to use an RTL language or not
```

Comment:
In PFE, we keep a list of RTL languages and check the stored locale against that.
https://github.com/zooniverse/Panoptes-Front-End/blob/2f7c55797bc9837ae5f0e6407d7c28d78b7bbe9b/app/redux/ducks/translations.js#L57-L60

A CSS selector like `html[lang=he]` handles most of the basic project display though.

Reply:
ooh, there's also https://developer.mozilla.org/en-US/docs/Web/CSS/:lang (didn't know about this one)

Reply:
That's new to me too. It'd be really nice if we didn't have to maintain a big list of RTL language codes, but could let the browser do that for us.